### PR TITLE
feat: add address propose integration test

### PR DESCRIPTION
**Pull request by pgallik**

### CODE REVIEW

File: src/AddressRegistry.Api.BackOffice.IntegrationTests/AddressControllerProposeTests.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    using System.Net.Http.Json;
+    using System.Threading.Tasks;
+    using FluentAssertions;
+    using Infrastructure;
+    using Microsoft.AspNetCore.Mvc.Testing;
+    using Xunit;
+
+    public class AddressControllerProposeTests
+    {
+        private const string Uri = "v2/adres/acties/voorstellen";
```
Comment (ArneD) [inline src/AddressRegistry.Api.BackOffice.IntegrationTests/AddressControllerProposeTests.cs:13]: shouldn't this be `v2/adressen/...`

File: src/AddressRegistry.Api.BackOffice.IntegrationTests/AddressRegistry.Api.BackOffice.IntegrationTests.csproj [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\packages\Be.Vlaanderen.Basisregisters.Build.Pipeline\Content\Be.Vlaanderen.Basisregisters.Build.Pipeline.Settings.Test.props" />
+  <ItemGroup>
+    <None Remove="appsettings.json" />
+  </ItemGroup>
+  <ItemGroup>
+    <Content Include="appsettings.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+  </ItemGroup>
```
Comment (ArneD) [inline src/AddressRegistry.Api.BackOffice.IntegrationTests/AddressRegistry.Api.BackOffice.IntegrationTests.csproj:10]: Can this be merged?

File: src/AddressRegistry.Api.BackOffice.IntegrationTests/appsettings.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    "ServiceName": "address-registry-backoffice-api"
+  },
+
+  "Cors": [
+
+  ],
+
+  "BaseUrl": "https://api.staging-basisregisters.vlaanderen/",
+
+  "DetailUrl": "https://basisregisters.vlaanderen.be/api/v1/adressen/{0}",
```
Comment (ArneD) [inline src/AddressRegistry.Api.BackOffice.IntegrationTests/appsettings.json:27]: probably v2?

**Review by ArneD — CHANGES_REQUESTED** [review]

File: src/AddressRegistry.Api.BackOffice.IntegrationTests/AddressControllerProposeTests.cs (hunk 1 above)
Comment (pgallik) [reply src/AddressRegistry.Api.BackOffice.IntegrationTests/AddressControllerProposeTests.cs:13]: ja

**Pull request merged**